manual/processed/2021-07-24-processed-manually.csv

```diff
@@ -1,13 +1,13 @@
 country,tests_cumulative,new_tests,tests_cumulative_corrected,new_tests_corrected,date,source,status,url
 Afghanistan,689412,0,NA,NA,2021-07-24,manually,pdf,http://covid.moph-dw.org/#/
-Barbados,NA,0,NA,NA,2021-07-24,manually,automated,https://gisbarbados.gov.bb/covid-19/
+Barbados,210425,0,NA,NA,2021-07-24,manually,automated,https://gisbarbados.gov.bb/covid-19/
 Benin,NA,0,NA,NA,2021-07-24,manually,automated,https://www.gouv.bj/coronavirus/
 Bhutan,NA,0,NA,NA,2021-07-24,manually,image,https://www.facebook.com/MoHBhutan/
-Bolivia,NA,0,NA,NA,2021-07-24,manually,image,https://www.minsalud.gob.bo/
+Bolivia,2041445,0,NA,NA,2021-07-24,manually,image,https://www.minsalud.gob.bo/
 Brazil,NA,0,NA,NA,2021-07-24,manually,automated,https://viz.saude.gov.br/extensions/DEMAS_C19Insumos_TESTES/DEMAS_C19Insumos_TESTES.html
-Brunei,NA,0,NA,NA,2021-07-24,manually,automated,http://www.moh.gov.bn/Lists/Latest%20news/AllItems.aspx
+Brunei,150053,0,NA,NA,2021-07-24,manually,automated,http://www.moh.gov.bn/Lists/Latest%20news/AllItems.aspx
 Cambodia,NA,0,NA,NA,2021-07-24,manually,automated,http://cdcmoh.gov.kh/
-Chile,60817,-18165403,60817,-18165403,2021-07-24,manually,automated,https://www.gob.cl/coronavirus/cifrasoficiales/
+Chile,NA,0,NA,NA,2021-07-24,manually,automated,https://www.gob.cl/coronavirus/cifrasoficiales/
```

Comment:
@Suthananthan this was correct, because the error is coming from the automation scrape process